lib/restforce/error_codes.rb

```diff
@@ -0,0 +1,184 @@
+# frozen_string_literal: true
+
+# Define error codes based on
+# https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_calls_concepts_core_data_objects.htm#statuscode
+module Restforce
+  module ErrorCode
+    class AllOrNoneOperationRolledBack < ResponseError; end
```

Comment:
I would be in favour of adding some kind of "lookup table" here (e.g. a hash which maps the `errorCode` to the class) to get rid of the `const_get` weirdness.

Reply:
I would make that hash lookup "fall back" to `const_get` if it fails.